lib/gameComponents/deck.dart

```diff
@@ -1 +1,30 @@
-class Deck {}
+import 'card.dart';
+import 'hand.dart';
+
+class Deck {
+  List<Hand> hands = [];
+  List<Card> cards = [];
```

Comment:
nit: To keep these properties from being modified from outside of the class, you may want to look at making these private (using an underscore before the name e.g. `_cards`, and creating a getter so that outside of the class can only read the property `List<Card> cards get => _cards;`. It can be a lot of boilerplate, but enforces consumers from doing anything they shouldn't!